package-lock.json

```diff
@@ -0,0 +1,1310 @@
+{
```

Comment:
This file needs to be removed too as our package-lock.jsons are in /front and /back